airbyte-integrations/connector-templates/source-java-jdbc/acceptance-test-config.yml

```diff
@@ -0,0 +1,7 @@
+# See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-{{dashCase name}}:dev
+tests:
```

Comment:
Please re-use the files of https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/source_acceptance_test_files/acceptance-test-config.yml.hbs#L1 . It has the latest configuration structure. I think it's fine to expose all the tests and let the connector developer decide which one they can enable.

Reply:
The original scope was to add only tests we already enabled for the existing sources. Because we are sure what exactly is required and how it will work for the Java sources.
Of course, it's not a big deal just put everything from the doc here as I do not need to work with it, but someone who wants to create a new source will have to do his investigation of how to make it work and what is missing.
I propose to add only what we can guarantee here.

Reply:
I'd suggest to comment the disabled test then. IMO it's developer's job to tweak the test config according to what's possible to run in the context of their connector.

Reply:
Done